fn/common.go

```diff
@@ -162,7 +164,19 @@ func writeTmpDockerfile(dir string, ff *funcfile) error {
 	}
 
 	// convert entrypoint string to slice
-	epvals := strings.Fields(*ff.Entrypoint)
+	bufferEp := stringToSlice(ff.Entrypoint)
+	bufferCmd := stringToSlice(ff.Cmd)
+
+	t := template.Must(template.New("Dockerfile").Parse(tplDockerfile))
+	err = t.Execute(fd, struct {
+		BaseImage, Entrypoint, Cmd string
+	}{rt, bufferEp.String(), bufferCmd.String()})
+	fd.Close()
```

Comment:
Should this `Close()` call be deferred up around ~165?  Though it not matter in the big picture, since something like a `Must()` `panic` would terminate the process anyways.